File: skills/astra-toolkit/clients/java/examples/admin-find-database.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.admin.AstraDBAdmin;
import com.datastax.astra.client.databases.definition.DatabaseInfo;
import java.util.UUID;

public class Example {
  public static void main(String[] args) {
    DataAPIClient client = new DataAPIClient(System.getenv("ASTRA_DB_APPLICATION_TOKEN"));

    AstraDBAdmin admin = client.getAdmin();

    DatabaseInfo databaseInfo = admin.getDatabaseInfo(UUID.fromString("**DATABASE_ID**"));

    System.out.println(databaseInfo);
  }
}

File: skills/astra-toolkit/clients/typescript/examples/admin-find-database.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

const client = new DataAPIClient(process.env.ASTRA_DB_APPLICATION_TOKEN!);

const admin = client.admin();

(async function () {
  const databaseInfo = await admin.dbInfo("**DATABASE_ID**");

  console.log(databaseInfo);
})();
